examples/build.rs

```diff
@@ -143,136 +143,145 @@ pub struct Cloud {
 
 fn main() -> anyhow::Result<()> {
     println!("cargo:rerun-if-changed=viam.json");
+    let use_nvs = match env::var("USE_NVS") {
```

Comment:
This is a legitimate build of the micro-rdk server, so I wouldn't say it's out of line. `USE_NVS` is intended to be a temporary measure, but it could use a better name. How do you feel about `MICRO_RDK_USE_NVS`?